### ISSUE TITLE
♿ AVT3 ♿ Overflow menu - User cannot navigate with arrow keys

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1590]
User tested with latest version of JAWS 2018, Windows 7, Chrome latest version

Screen reader user expected to be able to navigate the menu with arrow keys but can't. It was also not obvious how it works in general. If we're sticking on minimal controls we should announce them.

**Comment by dakahn** [comment]
Duplicate of #1233
